app/controllers/concerns/phone_confirmation_flow.rb

```diff
@@ -27,7 +27,15 @@ def this_send_confirmation_code_path
 
   def send_code
     send_confirmation_code
-    flash[:success] = t("notices.send_code.#{current_otp_method}")
+
+    if session[:code_sent].present?
+      flash[:success] = t(
+        "notices.send_code.#{current_otp_method}",
+          time: Time.current.strftime("%l:%M:%S%P %Z")
+      )
+    end
+
+    session[:code_sent] = 'true'
```

Comment:
should we remove this session var when OTP code get submitted successfully? It feels kinda sloppy to leave it, but also session data gets blown away on logout (every 8 mins of idleness) so maybe not a huge problem?